format/index.md

```diff
@@ -517,16 +514,21 @@ the following members:
 
 Examples of the supported formats:
 
-```
-"links": {
-  "self": "http://example.com/posts",
+```json
+{
```

Comment:
The JSON syntax highlighter requires a fully-formed JSON data structure, so I've added the `{` and `}` around the sample `"links"` section. (Same thing on lines 526-533.)